charset_normalizer/cd.py

```diff
@@ -62,7 +63,7 @@ def unicode_range_languages(primary_range: str) -> List[str]:
     return languages
 
 
-@lru_cache()
+@lru_cache(maxsize=None)
```

Comment:
As `maxsize` is None by default, do not set it plz.

Reply:
The default for `maxsize` is not `None`. It's 128. This is both in the docs and in the source for the functools module.

https://github.com/python/cpython/blob/8492b729ae97737d22544f2102559b2b8dd03a03/Lib/functools.py#L479